apps/system/js/update_manager.js

```diff
@@ -410,6 +410,32 @@
           nameDetails.classList.add('nosize');
         }
 
+        if (updatable.buildID) {
+          var buildId = document.createElement('span');
+          buildId.classList.add('font-light');
+          _localize(buildId, 'build-id', { buildid: updatable.buildID });
+          listItem.appendChild(buildId);
+        }
+
+        if (updatable.detailsURL && (updatable.detailsURL !== 'about:blank')) {
+          var detailsUrl = document.createElement('a');
+          _localize(detailsUrl, 'view-release-notes');
+          listItem.appendChild(detailsUrl);
+          listItem.onclick = (function(event) {
+            event.preventDefault();
```

Comment:
can you cover this with a test?
(asserting `defaultPrevented` and using `MockMozActivity` to assert that we launched the correct activity)